lib/rspec/sqlimit_equal.rb

```diff
@@ -0,0 +1,47 @@
+require "active_support/notifications"
+require "active_record"
+require "rspec"
+
+module RSpec
+  module SQLimitEqual
+    require_relative "sqlimit/counter"
+    require_relative "sqlimit/reporter"
+  end
+
+  Matchers.define :equal_query_limit do |expected|
```

Comment:
Let's call it `exec_queries` or smth like that ("equal" and "limit" should not be together).